### PR TITLE
Update Home.js

### PR DESCRIPTION
**Pull request by MayankPushpjeet**
Modified The CSS of Home.js File To make the Behaviour of Footer as expected.

### CODE REVIEW

File: src/Home.js [hunk 1]
```diff
@@ -114,10 +114,13 @@ const responsiveStyles = [
 
 const commonStyles = StyleSheet.create({
     container: {
+        display:'flex',
+        minHeight: '100%',
```
Comment (avikalpg) [inline src/Home.js:118]: Not sure if this is still required.

**Review by avikalpg — APPROVED** [review]
Looks good to me.

File: src/Home.js (hunk 1 above)
Comment (MayankPushpjeet) [reply src/Home.js:118]: yes they are not needed now.

Comment (avikalpg) [reply src/Home.js:118]: Please remove it and confirm whether the website still works.

**Comment by avikalpg** [comment]
Raised another PR #26 for this because @MayankPushpjeet was not responding to the PR comments.

**Pull request closed without merging**